Client/YooChooseNotifier.php

```diff
@@ -200,7 +222,7 @@ protected function getContentTypeId($contentId)
 
         try {
             $contentTypeId = $this->contentService->loadContentInfo($contentId)->contentTypeId;
-        } catch (\Exception $e) {
+        } catch (Exception $e) {
```

Comment:
Well, from system point of view, this smells like a DB inconsistency if you have Content that is missing Content Type. I think this should be logged. @andrerom what is your POV?